cpp/ql/test/query-tests/Likely Bugs/Format/NonConstantFormat/nested.cpp

```diff
@@ -0,0 +1,81 @@
+typedef void *va_list;
+#define va_start(ap, parmN)
+#define va_end(ap)
+#define va_arg(ap, type) ((type)0)
+#define NULL 0
+
+extern "C" int printf(const char *fmt, ...);
+extern "C" int snprint(char *buf, int len, const char *fmt, ...);
+extern "C" int _vsnprintf_s(
+   char *buffer,
+   int sizeOfBuffer,
+   int count,
+   const char *fmt,
+   va_list argptr
+);
+extern "C" int snprintf ( char * s, int n, const char * format, ... );
+
+struct A {
+  void do_print(const char *fmt0) {
+    char buf[32];
+    snprintf(buf, 32, fmt0);
```

Comment:
`// GOOD [this is just a wrapper, not the location of the error]`?

Reply:
GOOD, though there is no function wrapping involved.

Reply:
Actually, per our most recent discussion, I think we agreed that this is BAD precisely because it does not use a wrapper (I believe you brought up the example of `Log', which was a convincing one).